roles/commcare_sync/tasks/django.yml

```diff
@@ -102,11 +83,9 @@
 - name: Run Django collectstatic.
   become: yes
   become_user: "{{ app_user }}"
-  django_manage:
-    command: collectstatic
-    app_path: "{{ src_dir }}"
-    virtualenv: "{{ virtualenv_dir }}"
-    settings: "{{ django_settings_module }}"
+  command:
+    cmd: "{{ ansible_env.HOME }}/.local/bin/uv run manage.py collectstatic --no-input"
+    chdir: "{{ src_dir }}"
```

Comment:
The `django_manage` module needs a `runner` parameter that could be used like `runner: "{{ ansible_env.HOME }}/.local/bin/uv run manage.py`. Unfortunately I don't see a way to do that right now.